client/src/document/ingredients/browser-compatibility-table/index.scss

```
@@ -140,6 +152,32 @@
 .bc-version-label {
   display: none;
 }
+.expert .bc-version-label {
+  display: initial;
+}
+
+.expert .bc-icon-cell {
+  > .icon-wrap {
+    display: block;
+  }
+}
+
+.expert {
+  td {
+    &.bc-supports-no > span {
+      color: var(--text-primary-red);
+    }
+    &.bc-supports-partial > span {
+      color: var(--text-primary-yellow);
+    }
+    &.bc-supports-preview > span {
+      color: var(--text-primary-blue);
+    }
+    &.bc-supports-yes > span {
+      color: var(--text-primary-green);
+    }
```

Do you need to be that specific?

```suggestion
    &.bc-supports-no {
      color: var(--text-primary-red);
    }
    &.bc-supports-partial {
      color: var(--text-primary-yellow);
    }
    &.bc-supports-preview {
      color: var(--text-primary-blue);
    }
    &.bc-supports-yes {
      color: var(--text-primary-green);
    }
```

---

yes, that's for mobile otherwise the detailed history gets colored